### ISSUE TITLE
[Developer] Installation doesn't show "finished" screen after reboot

### ISSUE THREAD

**Issue by jahorton** [keymanapp/keyman#2182]
**Describe the bug**
If a user's system requires a reboot to fully install Keyman Developer, the post-reboot sequence of events doesn't indicate when the installation is complete.

**To Reproduce**
1. Have an older version of Keyman Developer installed
  a. If not previously installed, do so.
  b. If you just installed Developer the first time, reboot your computer (so that installed Developer dependencies will be loaded by the system)
2. Install the current version (12.0.50.0) of Keyman Developer.  This should trigger a "system reboot required" since some files will have been in use.
3. When the installer requests the reboot, click "OK" and let the system reboot.
4. ...
5. Once you log in to Windows post-reboot, a few seconds later the installer should auto-launch.  The screen shown will appear to be the installer's initial screen.
6. Tell it to install, checking for updates.
7. [The bug] Watch as the installer simply disappears.  It's actually done now (or quite shortly from now), but no more UI will be displayed.

**Expected behavior**
I'd expect to see a screen saying "Keyman Developer 12.0 installed successfully!" with an option to launch on the screen's close.  You know, standard 'finished installer' stuff.

**Environment**
OS: Windows 10 (Version 1903, build 18362.388)
Product:  Keyman Developer 12.0.50.0 (the first stable release)

**Comment by mcdurdin** [comment]
This is technically a feature rather than a bug -- the existing behaviour is by design, whether good or bad design is up for debate.

It may well be possible to eliminate the after-reboot installer process altogether, which would be better. Currently, the only reason that a reboot is required is if files are in use, which is nearly always Keyman Engine. With 15.0, we hope to eliminate this dependency, and that would make reboot so unlikely as to not warrant any change here.

So. We've lived with this for many versions of Keyman Developer, and I propose to postpone this once more, hopefully to eliminate the issue altogether in 15.0. 😄

**Comment by jahorton** [comment]
> This is technically a feature rather than a bug -- the existing behaviour is by design, whether good or bad design is up for debate.

As in, fixing the design would be considered an enhancement?

**Comment by mcdurdin** [comment]
Yes. I would prefer to eliminate the root issue, which is the reboot, than to spend time on resolving this otherwise very minor issue.
